packages/react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItem.styles.ts

```diff
@@ -157,6 +158,13 @@ export function getStyles(props: IPeoplePickerItemSelectedStyleProps): IPeoplePi
         },
       },
       selected && [
+        getFocusStyle(theme, {
+          inset: 2,
+          borderColor: 'transparent',
+          highContrastStyle: { left: 1, top: 1, bottom: 1, right: 1, outlineColor: 'ButtonText' },
```

Comment:
very minor nit: since we use `inset` above, it might be nice to use `inset: 1` here too

Reply:
done